tooling/cli/src/info/env_system.rs

```diff
@@ -20,7 +20,9 @@ struct VsInstanceInfo {
 const VSWHERE: &[u8] = include_bytes!("../../scripts/vswhere.exe");
 
 #[cfg(windows)]
-fn build_tools_version() -> crate::Result<Option<Vec<String>>> {
+fn build_tools_version() -> crate::Result<Vec<String>> {
+  use itertools::Itertools;
```

Comment:
not sure why you need this import?

Reply:
i used its `unique()` method to filter out duplicate VS version, in case users have both, a win10 and win11 sdk. I'll sort it and dedup it instead so we can get rid of that import :)

Fun fact: rust-analyzer actually imported it for `extend()` because i added that to the iterator by accident. Then i just went with that import for deduping the vec/iterator 🤷

Reply:
ah didn't see the `unique` usage, it is fine to keep then, I just thought it was accidentally imported or imported for `extend`

Reply:
> I just thought it was accidentally imported or imported for extend

You know me too well 😂